src/components/Modal.vue

```diff
@@ -304,108 +298,95 @@ Obs: Evite sobrepor modais sempre que possível.
 <template>
   <div>
     <vi-button primary @click="firstModalIsOpen = !firstModalIsOpen">Abrir alerta</vi-button>
-    <vi-modal
-      lock-scroll-background
-      title="Título da Modal"
-      title-size="2"
-      large-spacing
-      v-model="firstModalIsOpen"
-    >
+    <vi-modal lock-scroll-background title="Título da Modal" title-size="2" large-spacing v-model="firstModalIsOpen">
       <div slot="body">
         Exemplo de modal em uso
       </div>
       <div slot="footer">
         <vi-wrapper justify-content="flex-end">
-          <vi-button
-            outlined
-            dark
-            @click="firstModalIsOpen = false"
-          >Fecha</vi-button>
-          <vi-button
-            warning
-            @click="secondModalIsOpen = true"
-          >Exemplo 2</vi-button>
+          <vi-button outlined dark @click="firstModalIsOpen = false">Fecha</vi-button>
+          <vi-button warning @click="secondModalIsOpen = true">Exemplo 2</vi-button>
           <vi-button
             primary
-            @click="firstModalIsOpen = false; thirdModalIsOpen = true"
-          >Exemplo 3</vi-button>
+            @click="
+              firstModalIsOpen = false;
+              thirdModalIsOpen = true;
+            "
```

Comment:
A quebra não ficou tão legal também. Já que estamos mexendo aqui, né

Reply:
Não é possível alterar isso.

Reply:
Poxa, mas em todos os outros lugares fica certo. Por que aqui não? 🤔
Como consigo testar isso? Eu tentei editar o arquivo para uma forma mais amigável, rodei o linter e não tive problema algum. Será que fiz errado? Fiz conforme a figura abaixo:

![image](https://user-images.githubusercontent.com/767624/65463584-5bb59700-de2e-11e9-8ad5-a9b03bcfd633.png)

Reply:
O comentário parece estar no trecho errado. Ele vai quebrar o tag html desse jeito se não for alterado na mão. Feito.